test/selenium/src/lib/page/widget/info_widget.py

```diff
@@ -309,6 +363,7 @@ class Assessments(InfoWidget):
   """Model for Assessment object Info pages and Info panels."""
   # pylint: disable=invalid-name
   # pylint: disable=too-many-instance-attributes
+  # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
It's not a good practice to mute this pylint warning

Reply:
Done.